contract/area_types.go

```diff
@@ -24,6 +24,13 @@ type GetAreaTypeParentsRequest struct {
 	AreaType       string
 }
 
+type GetParentAreaCountRequest struct {
```

Comment:
If this is no longer being used can probably be removed

Reply:
:facepalm:

Reply:
this have been addressed.